invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -193,23 +193,31 @@ def get_languages(self, obj):
 
     def get_locations(self, obj):
         """Get locations."""
-        # FIXME: Add after UI support is there
-        # locations = []
-        # for location in obj['metadata'].get('locations', []):
-        #     location_string = ""
-        #     place = location.get('place')
-        #     if place:
-        #         location_string += f"name={place}"
-
-        #     geometry = location.get('geometry')
-        #     if geometry:
-        #         geo_type = geometry['type']
-        #         if geo_type == "Point":
-        #             coords = geometry['coordinates']
-        #             location_string += f"; lat={coords[0]}; lon={coords[1]}"
-
-        #     locations.append(location_string)
-        return missing
+        locations = []
```

Comment:
Isn't that similar if not the same with the marcxml schema method?

Reply:
It is, should I abstract it/make it one common method for all?

Reply:
I think that would be nice :) Less error-prone approach!